### ISSUE TITLE
ObjectMap is broken

### ISSUE THREAD

**Issue by TCreutzenberg** [libgdx/libgdx#5931]
#### Issue details
To start: Everything works fine with LibGDX 1.9.10 (just tested) and it worked fine with 1.9.11-SNAPSHOT until a few days ago, this definately is a "new" 1.9.11-SNAPSHOT problem.

My IDE just updated to the latest LibGDX 1.9.11-SNAPSHOT, and now my AssetLoader doesn't load some assets any more.
The tasks are empty, but the asset is nowhere to be found (checked via breakpoint-debugging before unloading the asset, after AssetManager.finishLoading()).

#### Reproduction steps/code
I would have to upload my whole repo which I don't want to actually, but I get the problem only with the (new) snapshot, everything worked well a few days ago.

#### Version of LibGDX and/or relevant dependencies
1.9.11-SNAPSHOT

#### Stacktrace
My stacktrace will not help but here it is anyway
```java
Exception in thread "LWJGL Application" com.badlogic.gdx.utils.GdxRuntimeException: Asset not loaded: config/worldMap.data
	at com.badlogic.gdx.assets.AssetManager.unload(AssetManager.java:232)
	at de.creutzenberg.towerdefence.framework.assets.TCAssetManager.unload(TCAssetManager.java:528)
	at de.creutzenberg.towerdefence.framework.Global.initializeAndUnloadGameFiles(Global.java:146)
	at de.creutzenberg.towerdefence.TCGame.initializeAndUnloadBasicStuff(TCGame.java:159)
	at de.creutzenberg.towerdefence.ScreenManager.switchToNextScreen(ScreenManager.java:204)
	at de.creutzenberg.towerdefence.framework.ATCScreen.render(ATCScreen.java:117)
	at de.creutzenberg.towerdefence.TCGame.render(TCGame.java:198)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:233)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:128)
```

#### Please select the affected platforms
Only tested and veryfied on:
- [X] Windows

**Comment by TCreutzenberg** [comment]
![AssetManagerBug](https://user-images.githubusercontent.com/17275393/75779044-3b80f100-5d59-11ea-9f86-2368c6174b95.PNG)

**Comment by mgsx-dev** [comment]
it might be the same issue as #5929 
FYI there was big changes in libgdx ObjectMap and such recently, some fixes are on the way.

**Comment by tommyettinger** [comment]
Working on it, this seems to be fine with the same fix as LongMap in #5930 . Change LoC for each file are small, hopefully this can be PRed OK.

**Comment by NathanSweet** [comment]
Fixed by @tommyettinger in #5932.

**Comment by TCreutzenberg** [comment]
Just wanted to confirm that the fix works for me. Thanks guys! 👍
